src/backend/src/services/reimbursement-requests.services.ts

```diff
@@ -73,4 +73,24 @@ export default class ReimbursementRequestService {
 
     return vendor.vendorId;
   }
+
+  /**
+   * Service function to create an expense type in our database
+   * @param name The name of the expense type
+   * @param code the expense type's code
```

Comment:
this should also be more specific, is this the Sabo code? does the club generate these codes?